src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> stock = new HashMap<>();

```

Comment:
Using a static `HashMap` for storing stock data can lead to potential issues with data consistency and thread safety. Consider using a more robust solution, such as dependency injection, to manage the storage instance, especially if the application is expected to run in a multi-threaded environment.